["Status changed:Running pre train steps","Status changed:Not started","Status changed:Pending:null:null","Model management server:https://kubernetes-internal-b60a62f4e1d24d21.elb.us-west-2.amazonaws.com/model-management","Model:datasource-trial-1","Version:jan30test","Downloading files for fitting","Python 3.5.4","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   239  100   239    0     0   4454      0 --:--:-- --:--:-- --:--:--  4509","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   683  100   683    0     0  12149      0 --:--:-- --:--:-- --:--:-- 12196","Status changed:Running:null:null","Downloading csv resource from https://s3-us-west-2.amazonaws.com/ukkhdeh3ptsk8kh3/transaction_w_short.csv for [fitting]","Start fitting","Traceback (most recent call last):","  File \"driver.py\", line 57, in <module>","    pipeline.fit(iterator.get_single())","  File \"/usr/local/lib/python3.5/site-packages/pipeline/pipeline.py\", line 32, in fit","    dataset = pipe.transform(dataset, supervised=True)","  File \"/usr/local/lib/python3.5/site-packages/pipeline/pipes_df.py\", line 78, in transform","    return df[self.states['columns_to_keep']]","  File \"/usr/local/lib/python3.5/site-packages/pandas/core/frame.py\", line 2133, in __getitem__","    return self._getitem_array(key)","  File \"/usr/local/lib/python3.5/site-packages/pandas/core/frame.py\", line 2177, in _getitem_array","    indexer = self.loc._convert_to_indexer(key, axis=1)","  File \"/usr/local/lib/python3.5/site-packages/pandas/core/indexing.py\", line 1269, in _convert_to_indexer","    .format(mask=objarr[mask]))","KeyError: \"['trans_date'] not in index\"","Unable to read output:output.pkl","Status changed:Not started","Status changed:Pending:null:null","Model management server:https://kubernetes-internal-b60a62f4e1d24d21.elb.us-west-2.amazonaws.com/model-management","Model:datasource-validation-1","Version:jan30test","Downloading files for fitting","Python 3.5.4","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   255  100   255    0     0   1601      0 --:--:-- --:--:-- --:--:--  1603","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   683  100   683    0     0  12559      0 --:--:-- --:--:-- --:--:-- 12418","Status changed:Running:null:null","Downloading csv resource from https://s3-us-west-2.amazonaws.com/ukkhdeh3ptsk8kh3/transaction_w_short_validation.csv for [fitting]","Start fitting","Traceback (most recent call last):","  File \"driver.py\", line 57, in <module>","    pipeline.fit(iterator.get_single())","  File \"/usr/local/lib/python3.5/site-packages/pipeline/pipeline.py\", line 32, in fit","    dataset = pipe.transform(dataset, supervised=True)","  File \"/usr/local/lib/python3.5/site-packages/pipeline/pipes_df.py\", line 78, in transform","    return df[self.states['columns_to_keep']]","  File \"/usr/local/lib/python3.5/site-packages/pandas/core/frame.py\", line 2133, in __getitem__","    return self._getitem_array(key)","  File \"/usr/local/lib/python3.5/site-packages/pandas/core/frame.py\", line 2177, in _getitem_array","    indexer = self.loc._convert_to_indexer(key, axis=1)","  File \"/usr/local/lib/python3.5/site-packages/pandas/core/indexing.py\", line 1269, in _convert_to_indexer","    .format(mask=objarr[mask]))","KeyError: \"['trans_date'] not in index\"","Unable to read output:output.pkl","Status changed:Not started","Status changed:Start training","Status changed:Pending:null:null","Model management server: https://kubernetes-internal-b60a62f4e1d24d21.elb.us-west-2.amazonaws.com/model-management","Model: LSTM-Bustout","Version: 2","Training ID: trainer-keras-LSTM-Bustout-2-cpu-be1e0f82","Training DS: datasource-trial-1","Validation DS: datasource-validation-1","Datasource ID: ","Downloading files for training","Python 3.5.2","Getting model from https://kubernetes-internal-b60a62f4e1d24d21.elb.us-west-2.amazonaws.com/model-managementmodel-management/KERAS/model/LSTM-Bustout/version/2/ placing in /input/model.json","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100  1673    0  1673    0     0   5531      0 --:--:-- --:--:-- --:--:--  5521","Getting training data source config file from https://kubernetes-internal-b60a62f4e1d24d21.elb.us-west-2.amazonaws.com/model-management/datasource/datasource-trial-1, placing in /input/training_datasource.json","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   239  100   239    0     0   1420      0 --:--:-- --:--:-- --:--:--  1431","Getting validation data source config file from https://kubernetes-internal-b60a62f4e1d24d21.elb.us-west-2.amazonaws.com/model-management/datasource/datasource-validation-1, placing in /input/validation_datasource.json","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   255  100   255    0     0   1522      0 --:--:-- --:--:-- --:--:--  1526","  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current","                                 Dload  Upload   Total   Spent    Left  Speed","  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0","100   683  100   683    0     0   4225      0 --:--:-- --:--:-- --:--:--  4242","Extracting the keras json from the model metadata ","Running driver file","Status changed:Running:null:null","Using TensorFlow backend.","Downloading csv resource from https://s3-us-west-2.amazonaws.com/ukkhdeh3ptsk8kh3/transaction_w_short.csv for [training]","Downloading csv resource from https://s3-us-west-2.amazonaws.com/ukkhdeh3ptsk8kh3/transaction_w_short_validation.csv for [validation]","Epoch 1/2","Traceback (most recent call last):","  File \"driver.py\", line 96, in <module>","    keras_model.fit_generator(training_iterator, steps_per_epoch=epoch_steps, epochs=int(nr_epoch))","  File \"/usr/local/lib/python3.5/dist-packages/keras/legacy/interfaces.py\", line 87, in wrapper","    return func(*args, **kwargs)","  File \"/usr/local/lib/python3.5/dist-packages/keras/models.py\", line 1227, in fit_generator","    initial_epoch=initial_epoch)","  File \"/usr/local/lib/python3.5/dist-packages/keras/legacy/interfaces.py\", line 87, in wrapper","    return func(*args, **kwargs)","  File \"/usr/local/lib/python3.5/dist-packages/keras/engine/training.py\", line 2115, in fit_generator","    generator_output = next(output_generator)","  File \"/usr/local/lib/python3.5/dist-packages/keras/utils/data_utils.py\", line 735, in get","    six.reraise(value.__class__, value, value.__traceback__)","  File \"/usr/local/lib/python3.5/dist-packages/six.py\", line 693, in reraise","    raise value","  File \"/usr/local/lib/python3.5/dist-packages/keras/utils/data_utils.py\", line 635, in data_generator_task","    generator_output = next(self._generator)","  File \"/usr/local/lib/python3.5/dist-packages/pipeline/pipeline.py\", line 95, in transformed_iterator","    dataset = pipe.transform(dataset, supervised=supervised)","  File \"/usr/local/lib/python3.5/dist-packages/pipeline/pipes_df.py\", line 78, in transform","    return df[self.states['columns_to_keep']]","  File \"/usr/local/lib/python3.5/dist-packages/pandas/core/frame.py\", line 2133, in __getitem__","    return self._getitem_array(key)","  File \"/usr/local/lib/python3.5/dist-packages/pandas/core/frame.py\", line 2177, in _getitem_array","    indexer = self.loc._convert_to_indexer(key, axis=1)","  File \"/usr/local/lib/python3.5/dist-packages/pandas/core/indexing.py\", line 1269, in _convert_to_indexer","    .format(mask=objarr[mask]))","KeyError: \"['trans_date'] not in index\"","Can not find performance json file:[performance.json]","Unable to read output folder:[output]","Execution finished"]